client/src/pages/SubmissionDetails.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <q-page>
+    <h2 class="q-pl-lg">
+      Manage: {{ submission.title }}
+    </h2>
+    <div id="q-app">
+      <div class="row q-col-gutter-lg q-pa-lg">
+        <section class="col-md-5 col-sm-6 col-xs-12">
+          <h3>Add a Reviewer</h3>
+          <div class="q-gutter-md row">
+            <q-select
+              v-model="model"
+              filled
+              use-input
+              hide-selected
+              fill-input
+              input-debounce="0"
+              hint="Minimum 2 characters to trigger filtering"
```

Comment:
Screen Reader for Chrome (ChromeVox) only announces the hint when this element is tabbed, which means Screen Reader should read something that more clearly describes its purpose.
```suggestion
              hint="Search for a user to assign."
```